static/app/views/discover/table/tableView.spec.tsx

```diff
@@ -357,27 +361,32 @@ describe('TableView > CellActions', function () {
           },
         })}
         isLoading={false}
-        projects={initialData.organization.projects}
         tableData={{
           data: [
             {
+              id: '1',
               title: '/random/transaction/name',
               'p99(measurements.custom.kibibyte)': 222.3,
               'p99(measurements.custom.kilobyte)': 444.3,
             },
           ],
+          // @ts-expect-error: MetaType is not compatible because we can't express the correct type
```

Comment:
is there nothing we can do to avoid adding `@ts-expect-error`?

Reply:
I tried a bunch of things but couldn't figure out how to tell TS what's going on.

Reply:
this was because the `columnType` in https://github.com/getsentry/sentry/blob/286347571c27aa18c28fbe177f0c66f0d040e47a/static/app/utils/discover/eventView.tsx#L65

The type wasn't working properly